lib/exchanges/bitfinex.js

```diff
@@ -44,28 +31,25 @@ function openWebSocket(pair) {
     "event": "subscribe",
     "channel": "book",
     "pair": pair,
-    "prec": "R0",
+    "prec": "P0",
     "len":"25"
   });
   ws.on('open', () => {
     ws.send(msg);
   });
 
-  var counter = 3;
+  var flag = true;
 
-  // TODO: Can we use the flags parameter instead of this counting system?
```

Comment:
I looked through it and I couldn't find anything obvious that would help us differentiate, I'll add the comment back in so other people can take a look.